dashboard/cron.py

```diff
@@ -403,14 +418,66 @@ def update_term(self):
 
         return status
 
+
+    # update course records with data returned from verify_course_ids, only making changes when necessary
+    def update_course(self, warehouse_courses_data):
+        # cron status
+        status = ""
+        logger.debug("while using verify_course_ids data to update course table")
+
+        logger.info(warehouse_courses_data.to_json())
```

Comment:
might use debug if the json data gets too big.

Reply:
Thanks! I changed to `debug` in this [commit](https://github.com/tl-its-umich-edu/my-learning-analytics/pull/666/commits/a0fa6b6d9f37bdbec3c7cf604f53a9949ab43af4).